TWTValidation/JSON Validator/AST Nodes/TWTJSONSchemaAmbiguousASTNode.m

```diff
@@ -39,4 +39,30 @@ - (void)acceptProcessor:(id<TWTJSONSchemaASTProcessor>)processor
     [processor processAmbiguousNode:self];
 }
 
+- (NSArray *)childrenReferenceNodes
+{
+    NSMutableArray *nodes = [[super childrenReferenceNodes] mutableCopy];
+    [nodes addObjectsFromArray:[self childrenReferenceNodesFromNodeArray:self.subNodes]];
+
+    return nodes;
+}
+
+
+- (TWTJSONSchemaASTNode *)nodeForPathComponents:(NSArray *)path
+{
+    TWTJSONSchemaASTNode *node = [super nodeForPathComponents:path];
+    if (node) {
+        return node;
+    }
+
+    for (TWTJSONSchemaASTNode *subnode in self.subNodes) {
+        node = [subnode nodeForPathComponents:path];
+        if (node) {
+            break;
```

Comment:
You could return here and make the return on line 65 return nil. That makes it clearer that if you get to that last statement, it’s because you didn’t find anything.